src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,152 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  private String genre;
+
+  @Column(nullable = false)
+  private String author;
+
+  @Column(nullable = false)
+  private String image;
+
+  @Column(nullable = false)
+  private String title;
+
+  @Column(nullable = false)
+  private String subtitle;
+
+  @Column(nullable = false)
+  private String publisher;
+
+  @Column(nullable = false)
+  private String year;
+
+  @Column(nullable = false)
+  private String pages;
+
+  @Column(nullable = false)
+  private String isbn;
+
+  public Book() {}
+
+  public Book(String genre, String author, String image, String title, String subtitle,
+      String publisher, String year, String pages, String isbn) {
+    this.genre = genre;
+    this.author = author;
+    this.image = image;
+    this.title = title;
+    this.subtitle = subtitle;
+    this.publisher = publisher;
+    this.year = year;
+    this.pages = pages;
+    this.isbn = isbn;
+  }
+
+  public long getId() {
+    return id;
+  }
+
+  public void setId(long id) {
+    this.id = id;
+  }
```

Comment:
This setter should not be included to avoid unwanted changes of the autogenerated value 🙂